setup.py

```diff
@@ -28,6 +28,7 @@
     install_requires=[
         'snowballstemmer',
         'six',
+        'configparser;python_version<"3.3"',
```

Comment:
This will break on old enough versions of pip and setuptools which, if I remember correctly, is anyone using Ubuntu between 14.04 and 16.04

Reply:
@Nurdok and I were just discussing it. I saw [this](https://hynek.me/articles/conditional-python-dependencies/) and thought about doing it in the `setup.py`, but it's reeeeeeeeeeeeally gross.  Have you encountered that issue before and have an example / boilerplate? 😜

Reply:
Bump, @sigmavirus24 ?